docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -124,6 +124,11 @@ Updates to indicate you're running your experimental version. You can uninstall
 your version and go back to the originally installed version by choosing your
 version and clicking Uninstall.
 
+If you only install the VSIX, then the IDE will behave correctly (ie. new compiler
+and IDE behavior), but the Build operation or building from the command-line won't. 
+To fix that, add a nuget package reference in your csproj to the 
```

Comment:
reading this without background info, I couldn't understand it. Add what package reference to what csproj?

Reply:
@alrz Thanks for the feedback. I've added some details and pictures. Let me know if it's better now

Reply:
Change of wording made it a lot clearer. Thanks!